packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -209,6 +204,15 @@ export class FastSync extends (EventEmitter as {new (): InitialSyncEventEmitter}
     }
   };
 
+  private getPeerReputations(): IReputation[] {
```

Comment:
Should this live in network module? It would be nice to store peer reputations/scores in metadata store.

Maybe separate PR?

Reply:
Well it's kinda specific to sync, as it gets only reputations from peers supporting sync protocols